#!/usr/bin/env python
PACKAGE = "refresh_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mission_file",     str_t,    1, "Path to the mission behaviro tree file", "")
gen.add("tick_frequency",   double_t, 0, "Frequency (Hz) at which the behavior tree is ticked", 30, 0.1, 500)

exit(gen.generate(PACKAGE, "refresh_ros", "TaskBehaviortreeEngine")) 
